iothub/service/src/Amqp/AmqpSessionHandler.cs

```diff
@@ -35,25 +36,25 @@ protected internal AmqpSessionHandler()
         /// upload notifications or receiving feedback messages.
         /// </summary>
         /// <param name="linkAddress">The link address to receive from/send to.</param>
-        /// <param name="connectionLossHandler">The handler to invoke if this session or its links are dropped.</param>
+        /// <param name="sessionLossHandler">The handler to invoke if this session or its links are dropped.</param>
```

Comment:
Should the comment still mention dropped links?

Reply:
Yes because this handler is still eventually invoked if its links are dropped. A link drop triggers the session to close.